packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4998,6 +5009,29 @@ class EditableTextState extends State<EditableText>
     if (selectionOverlay.toolbarIsVisible) {
       hideToolbar(hideHandles);
     } else {
+      if (_hideSystemToolbarLastTimestamp != null &&
+          (SchedulerBinding.instance.currentSystemFrameTimeStamp -
+                  _hideSystemToolbarLastTimestamp!) <
+              _systemToolbarToggleDebounceThreshold) {
```

Comment:
So this fix won't be applied when the system context menu isn't being used because _hideSystemToolbarLastTimestamp will be null :+1: .